src/pages/[platform]/build-a-backend/more-features/interactions/set-up-interactions/index.mdx

```diff
@@ -28,3 +28,15 @@ import gettingStartedJs from '/src/fragments/lib/interactions/js/getting-started
     'react-native': gettingStartedJs
   }}
 />
+
+<InlineFilter filters={['react-native']}>
+
+## Known Issues
+
+you may encounter the following error when starting the bundler:
```

Comment:
@renebrandel either we need to capitalize this Y or this may be a copy past error and should be?

<InlineFilter filters={['react-native']}>

## Known Issues

When importing alternative service providers listed below, instead of the default Pinpoint provider:
* Kinesis (`aws-amplify/analytics/kinesis`)
* Kinesis Data Firehose (`aws-amplify/analytics/kinesis-firehose`)
* Personalize Event (`aws-amplify/analytics/personalize`)

you may encounter the following error when starting the bundler:

> Error: Unable to resolve module stream from /path/to/node_modules/@aws-sdk/...
This is a [known issue](https://github.com/aws/aws-sdk-js-v3/issues/4877). Please follow [the steps](https://github.com/aws/aws-sdk-js-v3/issues/4877#issuecomment-1656007484) 
outlined in the issue to resolve the error.

</InlineFilter>

Reply:
resovled on slack